apps/verticalhome/js/pin_apps.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+(function (exports) {
+
+  function PinAppItem(index) {
+    if (index === null || index === undefined){
+      return false;
+    }
+    this.index = index;
+    //creating pin app element
+    var pinElem = document.createElement('div');
+    pinElem.className = 'pin-app-item';
+    pinElem.setAttribute('data-index', this.index);
+    pinElem.innerHTML = "<img class='pin-app-icon'><br></img>" +
+            "<span class='title'></span>";
```

Comment:
A <br> inside of an <img> doesn't make sense, does it? This use of innerHTML is maybe okay because it is a string literal. But better to just create the img and span elements explicitly. Then you won't have to query them below.